ui-kit/src/main/res/drawable/background_counter_new.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid android:color="@color/night" />
+    <corners android:radius="10dp" />
```

Comment:
Please, check one more time, we shouldn't have such radiuses. 
it should be 8 or 12

Reply:
✔